### DISCUSSION TITLE
챕터8. 웹 프로토콜 최적화

### DISCUSSION THREAD

**Discussion by Tanney-102** [Frontend-Divers/performance-optimization#36] [함께 얘기하고 싶은 내용]
.

**Comment by Tanney-102** [comment]
http2의 스트림 구조 덕분에 멀티플렉싱이 가능하다는 게 이해가 잘 안됩니다ㅠㅠ

**Comment by Tanney-102** [comment]
http2에서 제공하는 서버푸시를 활용한다면 웹소켓도 대체할 수 있을까?

이에 대해 [스택오버플로우](https://stackoverflow.com/questions/28582935/does-http-2-make-websockets-obsolete)에 올라온 내용이 있음. (아직 읽기 전)

**Reply by ritajeong** [reply]
HTTP2 푸시의 죽음 -부분을 보면 도움이 될 것 같아요. 
https://almanac.httparchive.org/en/2021/http

**Comment by Tanney-102** [comment]
헤더 압축은 진짜 좋은듯..
